src/components/ArticlePreview/ArticlePreview.jsx

```diff
@@ -2,25 +2,47 @@ import React from 'react';
 import { Link } from 'gatsby';
 import { renderRichText } from 'gatsby-source-contentful/rich-text';
 
-import * as styles from './ArticlePreview.module.scss';
+import { ArticleAuthor } from '../ArticleAuthor/ArticleAuthor';
+import { createBemBlockBuilder } from '../../utils';
+
+import './ArticlePreview.scss';
+
+const getBlocksWithArticleList = createBemBlockBuilder(['article-preview-list']);
+const getBlocksWithArticleItem = createBemBlockBuilder(['article-preview-item']);
 
 export const ArticlePreview = ({ posts }) => {
   if (!posts) return null;
   if (!Array.isArray(posts)) return null;
 
   return (
     <div className="container">
-      <ul className={styles.articleList}>
+      <ul className={getBlocksWithArticleList()}>
         {posts.map(post => {
           return (
-            <li key={post.id}>
-              <Link to={`/blog/${post.id}`} className={styles.link}>
-                <h2 className={styles.title}>{post.title.title}</h2>
+            <li className={getBlocksWithArticleItem()} key={post.id}>
+              <Link
+                to={`/blog/${post.slug}`}
+                className={getBlocksWithArticleItem('__content__link')}
```

Comment:
According to BEM class cannot have multiple `__` parts. Please update all classes you created. This will make the code consistent with other components
```suggestion
                className={getBlocksWithArticleItem('__content-link')}
```

Reply:
@ViktorSoroka07 it makes sense, I only refactored the CSS to create the bem elemens class, correctly